stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/controller/ChannelControllerImpl.kt

```diff
@@ -1046,10 +1047,13 @@ internal class ChannelControllerImpl(
                 event.watcherCount?.let { setWatcherCount(it) }
             }
             is ReactionNewEvent -> {
-                upsertEventMessage(event.message)
+                upsertMessage(event.message)
```

Comment:
We don't need to update the latest message so `upsertMessage ` instead of `upsertEventMessage `